<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Jogo Velocidade Furiosa</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>79</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="questioLable">
          <property name="text">
           <string>Pergunta aqui</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelplayerPositionsLabel">
          <property name="text">
           <string>Posição: 0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="optionAButton">
          <property name="text">
           <string>Opção 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="optionBButton">
          <property name="text">
           <string>Opção 2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="optionCButton">
          <property name="text">
           <string>Opção 3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="optionDButton">
          <property name="text">
           <string>Opção 4</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="rollDiceButton">
      <property name="text">
       <string>Lançar Dado</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
